care/facility/migrations/0393_rename_dosage_prescription_base_dosage_and_more.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 4.2.5 on 2023-10-30 09:52
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("facility", "0392_alter_dailyround_consciousness_level"),
+    ]
+
+    def copy_prn_to_dosage_type(apps, schema_editor):
+        Prescription = apps.get_model("facility", "Prescription")
+        Prescription.objects.filter(is_prn=True).update(dosage_type="PRN")
+
+    def reverse_copy_prn_to_dosage_type(apps, schema_editor):
+        Prescription = apps.get_model("facility", "Prescription")
+        Prescription.objects.filter(dosage_type="PRN").update(is_prn=True)
```

Comment:
We will have titrated prescriptions classified as normal prescriptions if we reverse this migration. Does this seem fine @sainak ?